docs/Microservices/RPS/securityRPS.md

```diff
@@ -32,20 +33,23 @@ This is the password that is used to encrypt the provisioning certificate .pfx f
 RPS fetches the password from Vault and will use it when it is needed to decrypt a provisioning certificate.
 
 ### 5 Device configuration information sent to Intel® AMT device
-This data is a set of information that Intel® AMT firmware will use to establish trust and then activate the Intel® AMT device.  Contained in this information is the hashed remote admin password and the MEBX password.  It is important to protect this set of information while it is being used by RPS and while in transit to the Intel® AMT device.  Ensuring that a secure (TLS encrypted) WebSocket connection is used when RPS is communicating with the client device will protect this data while in transit.  This set of information uses nonces to prevent replay of this data.
+This data is a set of information that Intel® AMT firmware will use to establish trust and then activate the Intel® AMT device.  Contained in this information is the hashed remote admin password and the MEBX password.  It is important to protect this set of information while it is being used by RPS and while in transit to the Intel® AMT device.  Ensuring that a secure (TLS encrypted) WebSocket connection is used when RPS is communicating with the client device will protect this data while in transit.  This set of information utilizes nonces to prevent replay of this data.
+
+### 6 MPS CIRA Login Credentials
+In order for an Intel&reg; AMT device to connect to the MPS over a CIRA connection, it needs to provide the correct login credentials for MPS.  These credentials are specified as part of the AMT Profile created in RPS.  When a device is configured by RPS, the MPS CIRA credentials will be sent to MPS using the Devices POST API call where MPS will store the credentials.  These credentials are verfied by the MPS when the CIRA connection is established.
```

Comment:
To connect to the MPS over a CIRA connection, the Intel&reg; AMT device needs to provide the correct login credentials for MPS.  These credentials are specified as part of the AMT Profile created in RPS.  When a device is configured by RPS, the MPS CIRA credentials will be sent to MPS using the Devices POST API call where MPS will store the credentials.  These credentials are verfied by the MPS when the CIRA connection is established.

Reply:
updated